# A standard bid message

string robotName # Name of the bidder
int32 taskID # Task it is bidding on

# Bid data
float64 penalty #calculated penalty
float64 time #Calculated time
float64 eff #calculated efficiency